### DISCUSSION TITLE
Customize toolbar

### DISCUSSION THREAD

**Discussion by BenPseudo** [xdan/jodit#896] [Q&A] [answered]
How do you customize the toolbar (buttons)? Choose specific buttons instead of having all of them?

**Comment by ProjectLinde37** [comment]
Same here on version 4.6.2

**Comment by LucienLeMagicien** [comment, marked as answer]
You can find the relevant doc here: https://xdsoft.net/jodit/docs/options.html#buttons
If you disable a plugin, it also doesn't register it's buttons: https://xdsoft.net/jodit/docs/options.html#disableplugins
The Playground allows you to select which buttons and plugins you want and generates a config for you: https://xdsoft.net/jodit/play.html?currentTab=Buttons

To configure the paragraph, font, etc dropdowns, you might need to use `Jodit.atom` as referenced here (otherwise, your config gets merged into the default one): https://github.com/xdan/jodit/issues/1110 -> https://xdsoft.net/jodit/pro/docs/how-to/add-custom-font-family.md
